package.json

```diff
@@ -17,7 +19,10 @@
     "eslint-config-brightspace": "latest",
     "eslint-plugin-html": "latest",
     "eslint-plugin-lit": "latest",
-    "polymer-cli": "latest"
+    "mocha": "latest",
+    "polymer-cli": "latest",
+    "puppeteer": "latest",
+    "visual-diff": "dbatiste/visual-diff"
```

Comment:
We'll need to decide where to put dev testing/demo tools.  We could keep them in `core` but it tends to bloat the `package.json` dependencies.  For the time being I've dropped this into my personal repo, but would good to be reviewed at some point.